### DISCUSSION TITLE
Missing GFM footnotes support

### DISCUSSION THREAD

**Discussion by johnny243** [Milkdown/milkdown#444] [General]
It seems footnotes are not working properly. Take the following example from https://www.markdownguide.org/extended-syntax/#footnotes:

```markdown
Here's a simple footnote,[^1] and here's a longer one.[^bignote]

[^1]: This is the first footnote.

[^bignote]: Here's one with multiple paragraphs and code.

    Indent paragraphs to include them in the footnote.

    `{ my code }`

    Add as many paragraphs as you like.
```

Will throw an error in the console:

![image](https://user-images.githubusercontent.com/5891646/160734539-2f6ce14a-3c93-441f-b027-f8bda1cb87e3.png)

It should be possible to render this, since `milkdown` uses https://github.com/remarkjs/remark-gfm under the hood. What am I missing?

**Comment by Saul-Mirone** [comment]
It's not supported yet. But plan to be supported in next version.

**Reply by johnny243** [reply]
Good to know! Thanks for the quick response 🙂

**Comment by Saul-Mirone** [comment]
https://github.com/Saul-Mirone/milkdown/issues/445

**Comment by Saul-Mirone** [comment]
Supported now.
![image](https://user-images.githubusercontent.com/10047788/161378370-1f673869-4d0f-4c1d-8f59-af024baf48ac.png)
